.github/CODEOWNERS

```diff
@@ -15,39 +15,128 @@
 /docs/ @grafana/docs-squad
 /contribute/ @grafana/docs-squad
 /docs/sources/developers/plugins/ @grafana/docs-squad @grafana/plugins-platform-frontend @grafana/plugins-platform-backend
-/docs/sources/developers/plugins/backend @grafana/docs-squad @grafana/plugins-platform-backend
+/docs/sources/developers/plugins/backend/ @grafana/docs-squad @grafana/plugins-platform-backend
 # Set up, dashboards/visualization, best practices: Chris Moyer
 # Alerting: Brenda Muir
 /docs/sources/administration/ @Eve832 @GrafanaWriter
-/docs/sources/alerting @brendamuir
-/docs/sources/best-practices/ @chri2547
+/docs/sources/alerting/ @brendamuir
 /docs/sources/dashboards/ @chri2547
 /docs/sources/datasources/ @Eve832 @GrafanaWriter
-/docs/sources/enterprise/ @Eve832 @GrafanaWriter
 /docs/sources/explore/ @Eve832 @GrafanaWriter
 /docs/sources/getting-started/ @chri2547
-/docs/sources/old-alerting @brendamuir
-/docs/sources/panels/ @chri2547
+/docs/sources/old-alerting/ @brendamuir
 /docs/sources/release-notes/ @Eve832 @GrafanaWriter
 /docs/sources/setup-grafana/ @chri2547
-/docs/sources/visualization/ @chri2547
 /docs/sources/whatsnew/ @Eve832 @GrafanaWriter
 
+# Backend code (wildcard)
+#/pkg/services/ @grafana/backend-platform
```

Comment:
i saw that you overwrite the subfolders inside of services later, maybe we should remove here, since accesscontrol/alerting/auth/authn inside of services are not ours

Reply:
just saw you commented it out 👯

Reply:
No, thanks, good catch. It's an accidental leftover